src/components/IGVFileSelector.js

```diff
@@ -1,10 +1,14 @@
 import _ from 'lodash/fp';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { div, h } from 'react-hyperscript-helpers';
 import { AutoSizer, List } from 'react-virtualized';
 import ButtonBar from 'src/components/ButtonBar';
 import { ButtonPrimary, LabeledCheckbox, Link } from 'src/components/common';
 import IGVReferenceSelector, { addIgvRecentlyUsedReference, defaultIgvReference } from 'src/components/IGVReferenceSelector';
+import { Ajax } from 'src/libs/ajax';
```

Comment:
please import the specific Ajax subareas directly like DrsUriResolver

Reply:
[Done](https://github.com/DataBiosphere/terra-ui/pull/5191/commits/d342c2acfcd360c9bef9aabca2d6a1b4a4c4a7a4)!

Reply:
looks great :)